README.md

```diff
@@ -67,6 +66,16 @@ and learn more about the major components by taking a look at
  * [Components](docs/components.md) - Detailed look at all components
 
 
+# Community
```

Comment:
Keeping it to community for now - I'll also mention the community repo and code of conduct (which I'll also add to the main repo)